WORKSPACE

```diff
@@ -93,11 +93,16 @@ maven_jar(
 
 http_archive(
     name = "com_google_protobuf",
-    sha256 = "9510dd2afc29e7245e9e884336f848c8a6600a14ae726adb6befdb4f786f0be2",
-    urls = ["https://github.com/protocolbuffers/protobuf/archive/v3.6.1.3.zip"],
-    strip_prefix = "protobuf-3.6.1.3",
+    sha256 = "d82eb0141ad18e98de47ed7ed415daabead6d5d1bef1b8cccb6aa4d108a9008f",
+    strip_prefix = "protobuf-b4f193788c9f0f05d7e0879ea96cd738630e5d51",
+    # Commit from 2019-05-15, update to protobuf 3.8 when available.
+    url = "https://github.com/protocolbuffers/protobuf/archive/b4f193788c9f0f05d7e0879ea96cd738630e5d51.tar.gz",
 )
 
+load("@com_google_protobuf//:protobuf_deps.bzl", "protobuf_deps")
+
+protobuf_deps()
```

Comment:
Can you update the getting started documentation since I assume if we merge this this will break all out downstream users when they update.

At least explain they need to add that to their build but also if possible link to any documentation that might exist on configuring that function (some such dependency functions have optional args).

Reply:
Good point.

protobuf_deps contains only zlib. I can inline its definition into `scala_repositories` so that it's transparent for end users.

Reply:
and the protobuf http_archive can be moved there, to simplify the getting started code.